pkgs/top-level/all-packages.nix

```diff
@@ -32571,6 +32571,8 @@ with pkgs;
 
   popura = callPackage ../tools/networking/popura { };
 
+  portmaster = callPackage ../by-name/po/portmaster {};
+
```

Comment:
This is no longer necessary with the `by-name` structure, so should be removed.

Reply:
Done